src/store/conditionsSlice.ts

```diff
@@ -42,8 +45,12 @@ const conditionsSlice = createSlice({
         state.push(action.payload);
       }
     },
+    setIsAssetSet(state, action: PayloadAction<string>) {
+      const index = parseInt(action.payload.split("-")[1]);
+      state[index].isAssetSet = true;
```

Comment:
what is index is NaN? ParseInt can return NaN

Reply:
and I guess that `parseInt()` can be replaced just with `Number(action.payload.split("-")[1])` or simply `+action.payload.split("-")[1]`